src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -12,48 +13,45 @@
 import java.util.NoSuchElementException;
 
 @RestController
```

Comment:
해당 컨트롤러가 전부 ReqeustMapping은 아닐텐데, 이렇게 제한을 주는 것보다는
```suggestion
@RestController("/lines")
```
로 해주는게 좋을 것 같습니다!